product_docs/docs/pgd/5/security/roles-and-replication.mdx

```diff
@@ -0,0 +1,49 @@
+---
+title: Roles and replication
+---
+
+DDL changes executed by a user are applied as that same user on each node.
+
+DML changes to tables are replicated as the table-owning user on the target node.
+We recommend but don't enforce that a table be owned by the same user on each node.
+
+Suppose table A is owned by user X on node1 and owned by user Y on node2. If user Y
+has higher privileges than user X, this might be viewed as a privilege escalation.
+Since some nodes have different use cases, we allow this but warn against it
```

Comment:
I notice that this chapter talks about "we" a lot, while it should likely be "PGD" in most cases (according to the [Documentation Styleguide](https://www.enterprisedb.com/docs/community/contributing/styleguide/#person) - should this be corrected as part of this section revamp exercise?

```suggestion
Since some nodes have different use cases, PGD allows this but warns against it
```